redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java

```diff
@@ -438,7 +442,10 @@ public void operationComplete(Future<List<Map<String, String>>> future) throws E
 
     private String createAddress(String host, Object port) {
         if (host.contains(":") && !host.startsWith("[")) {
-            host = "[" + host + "]";
+        	String[] splittedHost= host.split(":");
+            host = "[" + applyNatMap(splittedHost[0]) + splittedHost[1] + "]";
```

Comment:
IPv6 format should be taken in account

Reply:
fixed on https://github.com/redisson/redisson/pull/2137